README.md

```diff
@@ -36,7 +36,72 @@ To run this problem on a python Turing machine, just call `make run`. To run
 this problem on a Vim Turing machine, just call `make run-vim-machine`. To see
 the Vim Turing machine without running it, call `make open-vim-machine`.
 
+So Vim did what? Wait. How does does it even?
+============================
+
+So you run this program, and it works. Great! So what happened? Well the most
+common thing you're going to see is `y$@"`. What this does is yank from the
+current cursor to the end of the line and then executes the default register as
+a macro. This allows us to encode motions in lines and then execute them. We
+then chain lines together by ending lines with moving to a mark, or a search
+result, and then yanking and executing that line.
+
+Using that nifty trick, we begin by yanking the first line and executing it.
+That then sets off our mark initialization. We then jump to various markers in
+the file and then mark those positions using `m<letter>` where `<letter>` is
+whatever we want to call that mark. Generally the first initial of whatever the
+thing we're marking is. Once everything is marked, we then begin the state
```

Comment:
I'd just say, "we search for _<letter>: in the code and then use that to mark the pointer".

Reply:
Updated.